src/responses.js

```diff
@@ -1,5 +1,6 @@
 // @flow
 const fs = require('fs');
+const { format } = require('util');
```

Comment:
namingthings: I think `format` in the global space might be too ambiguous, but it's currently not a problem.